src/python/foglamp/sending_process.py

```diff
@@ -189,7 +241,17 @@ class SendingProcess:
 
     }
 
-    def __init__(self):
+    def __init__(self, _mgt_name, _mgt_port, _mgt_address):
+        """
+
+        Args:
+            _mgt_name: Unique name that represents the microservice - Used by the Storage layer
```

Comment:
Strictly speaking this is not used by the storage layer, this is the name of your micro service or task